pytorch_lightning/trainer/data_loading.py

```diff
@@ -37,12 +37,12 @@ class TrainerDataLoadingMixin(ABC):
     # this is just a summary on variables used in this abstract class,
     #  the proper values/initialisation should be done in child class
     global_rank: int
-    shown_warnings: ...
+    shown_warnings:...
```

Comment:
also, pep8 is having some issues with this: https://github.com/PyTorchLightning/pytorch-lightning/pull/5720#issuecomment-770295218

Reply:
can you report it on YAPF?

Reply:
I do not see flake8 complaining about this change...